react-native/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": ["../react/.eslintrc.json", "../base/.eslintrc.json"],
```

Comment:
I think we concluded that globals at the tops of files was fine, Considering the problems env can cause. It's more explicit this way.